barcodes_gs1_label/wizard/options_wizard.py

```diff
@@ -26,18 +25,30 @@ class BarcodesGS1PrintOptionsWizard(models.TransientModel):
     show_price = fields.Boolean(string="Show price", default=True)
     show_price_currency = fields.Boolean(string="Show currency", default=True)
     label_copies = fields.Integer(string="Copies", default=1, required=True)
+
+    def _default_barcode_type(self):
+        return (
+            self.env["barcodes.gs1.label.options.config"]
+            .search([("default", "=", True)])
+            .barcode_type
+        )
+
     barcode_type = fields.Selection(
         string="Barcode type",
-        selection=[
-            ("gs1-128", "GS1-128"),
-            ("gs1-datamatrix", "GS1-Datamatrix"),
-            ("ean13-code128", "EAN13 + Code128"),
-            ("ean13", "EAN13"),
-        ],
+        selection=lambda x: x.env[
+            "barcodes.gs1.label.options.config"
+        ].BARCODE_TYPE_SELECTION,
         required=True,
-        default="ean13",
+        compute="_compute_barcode_type",
+        readonly=False,
+        default=_default_barcode_type,
```

Comment:
You don't need the `default` if you have a `compute`, you can compute the initial value from the compute itself.